### PR TITLE
add collector log extraction back into logging-dump script for 4.x

### PR DESCRIPTION
**Pull request by jcantrill**
add collector log extraction back into logging-dump script for 4.x

cc @nhosoi @richm

### CODE REVIEW

File: hack/logging-dump.sh [hunk 1]
```diff
@@ -149,6 +149,15 @@ get_pod_logs() {
...
     oc -n $ns logs $pod -c $container | nice xz > $logs_folder/$pod-$container.log.xz || oc logs $pod | nice xz > $logs_folder/$pod.log.xz || echo ---- Unable to get logs from pod $pod and container $container
   done
+
+  if [ "$collector_folder" == "$2" ]	
+  then	
+    collector=fluentd
+    if [[ "${pod}" =~ .*rsyslog.* ]] ; then
+      collector=rsyslog
+    fi
+    oc exec $1 -c $collector logs | nice xz >> $logs_folder/$pod.log.xz
```
Comment (richm) [inline hack/logging-dump.sh]: for rsyslog, add the `--all` flag.  Not sure if you can do that for fluentd `logs` command, if it will be ignored

Comment (nhosoi) [reply hack/logging-dump.sh]: It looks fluentd logs ignore any options other than -f, -h and --help.  So, it'd be safe to add --all here.

Comment (jcantrill) [reply hack/logging-dump.sh]: @richm @nhosoi where does `-all` get added?

Comment (richm) [reply hack/logging-dump.sh]: `oc exec $1 -c $collector logs --all | ...`

Comment (jcantrill) [reply hack/logging-dump.sh]: updated.

**Comment by richm** [comment]
rsyslog flake
/retest

**Pull request merged**